pyproject.toml

```diff
@@ -56,14 +56,18 @@ zh = [
 anthropic = [
     'anthropic'
 ]
+gemini = [
+    'google-generativeai'
+]
 
 all = [
     'langcheck[de]',
     'langcheck[en]',
     'langcheck[ja]',
     'langcheck[ja-optional]',
     'langcheck[zh]',
-    'langcheck[anthropic]'
+    'langcheck[anthropic]',
+    'langcheck[gemini]'
```

Comment:
Maybe it is better to use `[google]` here following `[anthropic]`?

Reply:
hmm I feel like `gemini` is more consistent actually, since we call the eval client `GeminiEvalClient` and not `GoogleEvalClient` - wdyt?

Reply:
Makes sense!!